src/main/python/lexicon/module_classes.py

```diff
@@ -2112,10 +2110,11 @@ def getabbreviation(self):
 # It also stores "Added Info" (estimated, uncertain, etc) characteristics for each slot,
 # forearm, and the hand config overall.
 class HandConfigurationModule(ParameterModule):
-    def __init__(self, handconfiguration, overalloptions, articulators, timingintervals=None, addedinfo=None):
+    def __init__(self, handconfiguration, overalloptions, articulators, timingintervals=None, phonlocs=None, addedinfo=None):
         self._handconfiguration = handconfiguration
         self._overalloptions = overalloptions
-        super().__init__(articulators, timingintervals=timingintervals, addedinfo=addedinfo)
+        print(phonlocs)
```

Comment:
@Harukaichii are you still using this or can it be deleted?

Reply:
oh! yea that can be deleted, good catch :)